pluginconfig/execute.go

```diff
@@ -25,11 +22,53 @@ type ExecutePluginConfig struct {
 	MessageVisibilityInterval time.Duration `json:"messageVisibilityInterval"`
 }
 
-type ObserverInfo struct {
-	// Writer indicates that the node can contribute by sending reports to the destination chain.
-	// Being a Writer guarantees that the node can also read from the destination chain.
-	Writer bool `json:"writer"`
+type ExecuteOffchainConfig struct {
```

Comment:
Whats the difference between this and `ExecutePluginConfig`? There is some overlap in values.

Reply:
This will get posted on-chain and will inform what ends up in `ExecutePluginConfig`

Reply:
Could you add some comments to the struct to this effect?

Reply:
I've ended up embedding this struct in the `ExecutePluginConfig` struct, which holds configuration not just from the offchain config but from the general OCR config.